### PR TITLE
Resolving the method argument error.

### PR DESCRIPTION
**Pull request by chanduvkp**
Remy, Getting below error while doing 'make test-units'

actual = apd.sanitize_fatality_entity(input_, expected)
E       TypeError: sanitize_fatality_entity() takes 1 positional argument but 2 were given

tests/core/test_apd.py:636: TypeError

!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!! xdist.dsession.Interrupted: stopping after 1 failures !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
============================================================== 1 failed, 52 passed, 4 warnings in 6.45 seconds ==============================================================
ERROR: InvocationError for command /Users/cvenigalla/projects/scrapd/.tox/py37/bin/pytest -x --junitxml=/tmp/pytest/junit-py37.xml --cov-report term-missing --cov-report html --cov=scrapd -m 'not integrations' tests (exited with code 2)
__________________________________________________________________________________ summary __________________________________________________________________________________
ERROR:   py37: commands failed

### CODE REVIEW

File: scrapd/core/apd.py [hunk 1]
```diff
@@ -260,7 +260,12 @@ def sanitize_fatality_entity(d):
...
     """
     # All values must be strings.
+    print('chandu')
     for k, v in d.items():
+        print('chandu')
+        print(type(v))
+        # if not v.strip():
+        #     continue
         if isinstance(v, list):
             d[k] = ' '.join(v)
```
Comment (rgreinho) [inline scrapd/core/apd.py]: I think you can remove the `print('chandu')` 😁

File: tests/core/test_apd.py [hunk 2]
```diff
@@ -606,3 +606,32 @@ def test_extract_twitter_description_meta_00(input_, expected):
...
+    (
+        {
+            "Time": None
+        },
+        {},
+    ),
+))
+def test_sanitize_fatality_entity(input_, expected):
+    """Ensure."""
+    actual = apd.sanitize_fatality_entity(input_, expected)
```
Comment (rgreinho) [inline tests/core/test_apd.py]: The call to the function is incorrect. 

It should be:

```python
actual = apd.sanitize_fatality_entity(input_)
```

**Comment by chanduvkp** [comment]
Hi Remy,

I am seeing below error:

============================================================== 1 failed, 51 passed, 4 warnings in 6.56 seconds ==============================================================
ERROR: InvocationError for command /Users/cvenigalla/projects/scrapd/.tox/py37/bin/pytest -x --junitxml=/tmp/pytest/junit-py37.xml --cov-report term-missing --cov-report html --cov=scrapd -m 'not integrations' tests (exited with code 2)
__________________________________________________________________________________ summary __________________________________________________________________________________
ERROR:   py37: commands failed

**Comment by rgreinho** [comment]
@chanduvkp the unit tests are failing. If you look just above this message, `pytest` tells you which test failed and you can work on fixing the right code.

File: tests/core/test_apd.py [hunk 3]
```diff
@@ -606,3 +606,35 @@ def test_extract_twitter_description_meta_00(input_, expected):
     """Ensure we can extract the twitter tittle from the meta tag."""
     actual = apd.extract_twitter_description_meta(input_)
     assert actual == expected
+
+
+@pytest.mark.parametrize('input_', (
```
Comment (rgreinho) [inline tests/core/test_apd.py]: This should be:

```python
@pytest.mark.parametrize('input_,expected', (
```

File: tests/core/test_apd.py [hunk 4]
```diff
@@ -606,3 +606,35 @@ def test_extract_twitter_description_meta_00(input_, expected):
...
+        {},
+    ),
+    (
+        {
+            "Time": None
+        },
+        {},
+    ),
+) )
+def test_sanitize_fatality_entity(input_):
```
Comment (rgreinho) [inline tests/core/test_apd.py]: This should be:

```python
def test_sanitize_fatality_entity(input_, expected):
```

File: tests/core/test_apd.py [hunk 5]
```diff
@@ -606,3 +606,35 @@ def test_extract_twitter_description_meta_00(input_, expected):
...
+    """Ensure."""
+    actual = apd.sanitize_fatality_entity(input_)
+    assert actual == ''
```
Comment (rgreinho) [inline tests/core/test_apd.py]: This should be:

```python
assert actual == expected
```

**Comment by rgreinho** [comment]
Fixed by #139.

**Pull request closed without merging**